### PR TITLE
FHAC-316:Remove Null checks for X12 RealTime,Batch Request and Response

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1. Remove Null checks for X12 RealTime,Batch Request and Response
2. Ensured Audit Logging is happening

### CODE REVIEW

File: Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/audit/transform/CORE_X12AuditDataTransform.java [hunk 1]
```diff
@@ -358,65 +352,6 @@ protected boolean areRequiredCOREX12BatchFieldsNull(Object msg, AssertionType as
         if (oXDR.areRequiredUserTypeFieldsNull(assertion)) {
```
Comment (jsmith2) [inline Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/audit/transform/CORE_X12AuditDataTransform.java]: I don't think we need the assertion required fields checked either.  If those are null, we still should pass it.

**Comment by sailajaa** [comment]
Reviewed PR and good with merge.

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Comment by nareshcgi** [comment]
retest this please

**Pull request merged**